contrib/platform/src/com/sun/jna/platform/win32/WinNT.java

```diff
@@ -1074,10 +1074,11 @@ public FILE_NOTIFY_INFORMATION next() {
     int REG_NOTIFY_CHANGE_ATTRIBUTES = 0x00000002;
     int REG_NOTIFY_CHANGE_LAST_SET = 0x00000004;
     int REG_NOTIFY_CHANGE_SECURITY = 0x00000008;
+    int REG_NOTIFY_THREAD_AGNOSTIC = 0x10000000;
 
     int REG_LEGAL_CHANGE_FILTER = REG_NOTIFY_CHANGE_NAME
             | REG_NOTIFY_CHANGE_ATTRIBUTES | REG_NOTIFY_CHANGE_LAST_SET
-            | REG_NOTIFY_CHANGE_SECURITY;
+            | REG_NOTIFY_CHANGE_SECURITY | REG_NOTIFY_THREAD_AGNOSTIC;
```

Comment:
This could be a breaking change if this constant is used in code before Windows 8.  Do you know if Windows 7 ignores this flag?

Reply:
Well, the newer Windows headers do include this new definition, so I guess it's appropriate to include.